pkg/processor/sourceprocessor/source_processor_test.go

```diff
@@ -39,16 +39,20 @@ func createConfig() *Config {
 	return config
 }
 
-var (
-	cfg = createConfig()
-
-	k8sLabels = map[string]string{
+func createK8sLabels() map[string]string {
```

Comment:
We need (not in this PR) to change the other shared `var`s into `func`s as well to prevent tests sharing variables

👍  for this change